www/templates/goals/inventory.html

```diff
@@ -0,0 +1,48 @@
+<ion-view view-title="Inventory">
+		<ion-content class="has-header" overflow-scroll="true" padding="false">
+		<div style="height:157px; position:relative;">
```

Comment:
if so, is it possible to separate it out into a separate commit so that we can revert other commits more easily later?